### ISSUE TITLE
[Tab Extended]: QA: User is unable to navigate to other tabs using arrow keys from the currently active tab

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9229]
### Description

Page Element: Arrow key navigation among the tab controls

Issue Description:
User is unable to navigate to other tabs using arrow keys from the currently active tab

Expected:
User should be able to navigate to other tabs using arrow keys from the currently active tab. Depending on the orientation, either the up/down or the Right/Left arrow keys can be used. The following APG link can be referred. 
https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html

User Impact:
When the standard keyboard navigation method is not supported, it impacts the user’s ability to interact with page controls. This affects screen reader and keyboard-only users. 

### Component(s) impacted

Web component Canary and Staging: Tabs extended Default Variant as well as Tabs extended - with media 

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component canary: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
